### PR TITLE
CTOR-967 : November release note

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

November Release note

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] 25.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

**Comment by lucie-dubrunfaut** [comment]
Actuellement il me manque les PR de packs de : 
- Mosquitto (PR Plugins + Pack)
- DataDomain (PR Pack + Documentation)

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md [hunk 1]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
...
+- [**Atrica Routeur**](../procedures/network-routers-atrica-snmp.md) - Added service discovery rule for connections mode.
+- [**Centreon Database**](../procedures/applications-monitoring-centreon-database.md) - Used the proper mariadb message when checking mysql replication state.
+- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Added a lot of enhancements leading to a breaking change. [MANQUE LA PR]
+- [**MySQL/MariaDB**](../procedures/applications-databases-mysql.md) - Used the proper mariadb message when checking mysql replication state.
+
+</TabItem>
+<TabItem value="Breaking changes" label="Breaking changes">
+
+- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Added a lot of enhancements leading to a breaking change. [MANQUE LA PR]
+- [**Linux Telegraf Agent**](../procedures/operatingsystems-linux-telegraf-agent.md) - Changed the inherited template to passive.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md:25]:
```suggestion
- [**Linux Telegraf Agent**](../procedures/operatingsystems-linux-telegraf-agent.md) - The default inherited templates are now passive.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md [hunk 2]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
...
+- [**PostgreSQL**](../procedures/applications-databases-postgresql.md) - Fixed exclude options working as includes.
+- [**Windows Telegraf Agent**](../procedures/operatingsystems-windows-telegraf-agent.md) - Changed the inherited template to passive.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md:27]:
```suggestion
- [**Windows Telegraf Agent**](../procedures/operatingsystems-windows-telegraf-agent.md) - The default inherited templates are now passive.
```

File: pp/integrations/plugin-packs/releases/release-notes.md [hunk 3]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
...
+- [**Atrica Routeur**](../procedures/network-routers-atrica-snmp.md) - Added service discovery rule for connections mode.
+- [**Centreon Database**](../procedures/applications-monitoring-centreon-database.md) - Used the proper mariadb message when checking mysql replication state.
+- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Added a lot of enhancements leading to a breaking change. [MANQUE LA PR]
+- [**MySQL/MariaDB**](../procedures/applications-databases-mysql.md) - Used the proper mariadb message when checking mysql replication state.
+
+</TabItem>
+<TabItem value="Breaking changes" label="Breaking changes">
+
+- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Added a lot of enhancements leading to a breaking change. [MANQUE LA PR]
+- [**Linux Telegraf Agent**](../procedures/operatingsystems-linux-telegraf-agent.md) - Changed the inherited template to passive.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/releases/release-notes.md:25]:
```suggestion
- [**Linux Telegraf Agent**](../procedures/operatingsystems-linux-telegraf-agent.md) - The default inherited templates are now passive.
```

File: pp/integrations/plugin-packs/releases/release-notes.md [hunk 4]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
...
+- [**PostgreSQL**](../procedures/applications-databases-postgresql.md) - Fixed exclude options working as includes.
+- [**Windows Telegraf Agent**](../procedures/operatingsystems-windows-telegraf-agent.md) - Changed the inherited template to passive.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/releases/release-notes.md:27]:
```suggestion
- [**Windows Telegraf Agent**](../procedures/operatingsystems-windows-telegraf-agent.md) - The default inherited templates are now passive.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md [hunk 5]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
...
+- [**Centreon Database**](../procedures/applications-monitoring-centreon-database.md) - Used the proper mariadb message when checking mysql replication state.
+- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Added a lot of enhancements leading to a breaking change. [MANQUE LA PR]
+- [**MySQL/MariaDB**](../procedures/applications-databases-mysql.md) - Used the proper mariadb message when checking mysql replication state.
+
+</TabItem>
+<TabItem value="Breaking changes" label="Breaking changes">
+
+- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Added a lot of enhancements leading to a breaking change. [MANQUE LA PR]
+- [**Linux Telegraf Agent**](../procedures/operatingsystems-linux-telegraf-agent.md) - The default inherited templates are now passive.
+- [**PostgreSQL**](../procedures/applications-databases-postgresql.md) - Fixed exclude options working as includes.
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md:26]:
```suggestion
- [**PostgreSQL**](../procedures/applications-databases-postgresql.md) - Fixed `exclude` option working as includes.
```

File: pp/integrations/plugin-packs/releases/release-notes.md [hunk 6]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
...
+- [**Centreon Database**](../procedures/applications-monitoring-centreon-database.md) - Used the proper mariadb message when checking mysql replication state.
+- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Added a lot of enhancements leading to a breaking change. [MANQUE LA PR]
+- [**MySQL/MariaDB**](../procedures/applications-databases-mysql.md) - Used the proper mariadb message when checking mysql replication state.
+
+</TabItem>
+<TabItem value="Breaking changes" label="Breaking changes">
+
+- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Added a lot of enhancements leading to a breaking change. [MANQUE LA PR]
+- [**Linux Telegraf Agent**](../procedures/operatingsystems-linux-telegraf-agent.md) - The default inherited templates are now passive.
+- [**PostgreSQL**](../procedures/applications-databases-postgresql.md) - Fixed exclude options working as includes.
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/releases/release-notes.md:26]:
```suggestion
- [**PostgreSQL**](../procedures/applications-databases-postgresql.md) - Fixed `exclude` option working as includes.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md [hunk 7]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
...
 ## 2024
 
+### Novembre
+
+<Tabs groupId="sync">
+<TabItem value="Enhancements" label="Enhancements">
+
+- [**Atrica Routeur**](../procedures/network-routers-atrica-snmp.md) - Added service discovery rule for connections mode.
+- [**Centreon Database**](../procedures/applications-monitoring-centreon-database.md) - Used the proper mariadb message when checking mysql replication state.
+- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Added several new modes and services discovery rules.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md:18]:
```suggestion
- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Added several new modes and service discovery rules.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md [hunk 8]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
...
+- [**MySQL/MariaDB**](../procedures/applications-databases-mysql.md) - Used the proper mariadb message when checking mysql replication state.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md:19]:
```suggestion
- [**MySQL/MariaDB**](../procedures/applications-databases-mysql.md) - Used the correct MariaDB message when checking MySQL replication state.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md [hunk 9]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
 
 ## 2024
 
+### Novembre
+
+<Tabs groupId="sync">
+<TabItem value="Enhancements" label="Enhancements">
+
+- [**Atrica Routeur**](../procedures/network-routers-atrica-snmp.md) - Added service discovery rule for connections mode.
+- [**Centreon Database**](../procedures/applications-monitoring-centreon-database.md) - Used the proper mariadb message when checking mysql replication state.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md:17]:
```suggestion
- [**Centreon Database**](../procedures/applications-monitoring-centreon-database.md) - Used the correct MariaDB message when checking MySQL replication state.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md [hunk 10]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
 
 ## 2024
 
+### Novembre
+
+<Tabs groupId="sync">
+<TabItem value="Enhancements" label="Enhancements">
+
+- [**Atrica Routeur**](../procedures/network-routers-atrica-snmp.md) - Added service discovery rule for connections mode.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md:16]:
```suggestion
- [**Atrica Routeur**](../procedures/network-routers-atrica-snmp.md) - Added service discovery rule for **connections** mode.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md [hunk 11]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
...
+- [**Centreon Database**](../procedures/applications-monitoring-centreon-database.md) - Used the proper mariadb message when checking mysql replication state.
+- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Added several new modes and services discovery rules.
+- [**MySQL/MariaDB**](../procedures/applications-databases-mysql.md) - Used the proper mariadb message when checking mysql replication state.
+
+</TabItem>
+<TabItem value="Breaking changes" label="Breaking changes">
+
+- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Changed modes name and dependencies paths.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md:24]:
```suggestion
- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Changed mode names and dependencies paths.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md [hunk 12]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
...
+- [**Linux Telegraf Agent**](../procedures/operatingsystems-linux-telegraf-agent.md) - The default inherited templates are now passive.
+- [**PostgreSQL**](../procedures/applications-databases-postgresql.md) - Fixed `exclude` option working as includes.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md:26]:
```suggestion
- [**PostgreSQL**](../procedures/applications-databases-postgresql.md) - Fixed `exclude` option working as "include".
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md [hunk 13]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
...
+- [**Windows Telegraf Agent**](../procedures/operatingsystems-windows-telegraf-agent.md) - The default inherited templates are now passive.
+
+</TabItem>
+<TabItem value="Fix" label="Fix">
+
+- [**Amazon CloudTrail**](../procedures/cloud-aws-cloudtrail.md) - Fixed command for event lookup from community feedback [PR 5086](https://github.com/centreon/centreon-plugins/pull/5086).
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md:32]:
```suggestion
- [**Amazon CloudTrail**](../procedures/cloud-aws-cloudtrail.md) - Fixed command for event lookup, from community feedback [PR 5086](https://github.com/centreon/centreon-plugins/pull/5086).
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md [hunk 14]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
...
+- [**Eclipse Mosquitto MQTT**](../procedures//applications-eclipse-mosquitto-mqtt.md) - Fixed ` warning-regexp` and `critical-regexp` options.
+- [**HPE Primera REST API**](../procedures/hardware-storage-hpe-primera-restapi.md) - Fixed status handling & retry on invalid token from community feedback [PR 5256](https://github.com/centreon/centreon-plugins/pull/5256).
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md:34]:
```suggestion
- [**HPE Primera REST API**](../procedures/hardware-storage-hpe-primera-restapi.md) - Fixed status handling & retry on invalid token, from community feedback [PR 5256](https://github.com/centreon/centreon-plugins/pull/5256).
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/releases/release-notes.md [hunk 15]
```diff
@@ -8,6 +8,36 @@ import TabItem from '@theme/TabItem';
...
+<TabItem value="Fix" label="Fix">
+
+- [**Amazon CloudTrail**](../procedures/cloud-aws-cloudtrail.md) - Fixed command for event lookup from community feedback [PR 5086](https://github.com/centreon/centreon-plugins/pull/5086).
+- [**Eclipse Mosquitto MQTT**](../procedures//applications-eclipse-mosquitto-mqtt.md) - Fixed ` warning-regexp` and `critical-regexp` options.
+- [**HPE Primera REST API**](../procedures/hardware-storage-hpe-primera-restapi.md) - Fixed status handling & retry on invalid token from community feedback [PR 5256](https://github.com/centreon/centreon-plugins/pull/5256).
+- [**Speedtest**](../procedures/applications-monitoring-speedtest.md) - Fixed packaging issue leading to conflict between speedtest and speedtest-cli binaries.
+- [**Cisco WLC**](../procedures/network-cisco-wlc-snmp.md) - Fixed wrong unit for roundtrip-time.
+
+</TabItem>
+</Tabs>
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/releases/release-notes.md:39]:
```suggestion
<Tabs groupId="sync">
<TabItem value="Enhancements" label="Enhancements">

- [**Atrica Routeur**](../procedures/network-routers-atrica-snmp.md) - Added service discovery rule for **connections** mode.
- [**Centreon Database**](../procedures/applications-monitoring-centreon-database.md) - Used the correct MariaDB message when checking MySQL replication state.
- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Added several new modes and service discovery rules.
- [**MySQL/MariaDB**](../procedures/applications-databases-mysql.md) - Used the correct MariaDB message when checking MySQL replication state.

</TabItem>
<TabItem value="Breaking changes" label="Breaking changes">

- [**EMC Data Domain SNMP**](../procedures/hardware-storage-emc-datadomain-snmp.md) - Changed mode names and dependencies paths.
- [**Linux Telegraf Agent**](../procedures/operatingsystems-linux-telegraf-agent.md) - The default inherited templates are now passive.
- [**PostgreSQL**](../procedures/applications-databases-postgresql.md) - Fixed `exclude` option working as "include".
- [**Windows Telegraf Agent**](../procedures/operatingsystems-windows-telegraf-agent.md) - The default inherited templates are now passive.

</TabItem>
<TabItem value="Fix" label="Fix">

- [**Amazon CloudTrail**](../procedures/cloud-aws-cloudtrail.md) - Fixed command for event lookup, from community feedback [PR 5086](https://github.com/centreon/centreon-plugins/pull/5086).
- [**Eclipse Mosquitto MQTT**](../procedures//applications-eclipse-mosquitto-mqtt.md) - Fixed ` warning-regexp` and `critical-regexp` options.
- [**HPE Primera REST API**](../procedures/hardware-storage-hpe-primera-restapi.md) - Fixed status handling & retry on invalid token, from community feedback [PR 5256](https://github.com/centreon/centreon-plugins/pull/5256).
- [**Speedtest**](../procedures/applications-monitoring-speedtest.md) - Fixed packaging issue leading to conflict between speedtest and speedtest-cli binaries.
- [**Cisco WLC**](../procedures/network-cisco-wlc-snmp.md) - Fixed wrong unit for roundtrip-time.

</TabItem>
</Tabs>
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**